pygerduty/__init__.py

```diff
@@ -414,6 +414,9 @@ def resolve(self, requester_id):
     def acknowledge(self, requester_id):
         self._do_action('acknowledge', requester_id=requester_id)
 
+    def snooze(self, duration):
+        self._do_action('snooze', duration=duration)
```

Comment:
Doh...so it does.  I was distracted by the shiny "Required=No" here:
https://developer.pagerduty.com/documentation/rest/incidents/snooze
:-)